installing/installing_aws/installing-aws-wavelength.adoc

```diff
@@ -0,0 +1,280 @@
+include::modules/aws-edge-zones-cluster-limitations.adoc[leveloffset=+2]
+
+// Edge compute pools and AWS {aws-service-name-full}
+include::modules/edge-machine-pools-aws-local-zones.adoc[leveloffset=+2]
+
+[role="_additional-resources"]
+.Additional resources
+
+* xref:../../networking/changing-cluster-network-mtu.adoc#mtu-value-selection_changing-cluster-network-mtu[Changing the MTU for the cluster network]
+* xref:../../networking/changing-cluster-network-mtu.adoc#nw-ovn-ipsec-enable_configuring-ipsec-ovn[Enabling IPsec encryption]
+* xref:../../nodes/scheduling/nodes-scheduler-taints-tolerations.adoc#nodes-scheduler-taints-tolerations-about_nodes-scheduler-taints-tolerations[Understanding taints and tolerations]
+
+
+[role="_additional-resources"]
+.Additional resources
+
+* xref:../../storage/understanding-persistent-storage.adoc#pvc-storage-class_understanding-persistent-storage[Storage classes]
+
+* xref:../../networking/ingress-sharding.html#nw-ingress-sharding_ingress-sharding[Ingress Controller sharding]
+
+== Prerequisites for the installation
+
+// Opting in to the {aws-service-name-short} Group
+include::modules/installation-aws-add-local-zone-locations.adoc[leveloffset=+2]
+
+// Internet access for OpenShift Container Platform
+include::modules/cluster-entitlements.adoc[leveloffset=+2]
+
+// Obtaining an AWS Marketplace image
+include::modules/installation-aws-marketplace-subscribe.adoc[leveloffset=+2]
+
+//Installing the OpenShift CLI by downloading the binary: Moved up to precede `ccoctl` steps, which require the use of `oc`
+include::modules/cli-installing-cli.adoc[leveloffset=+2]
+
+// Obtaining the installation program
+include::modules/installation-obtaining-installer.adoc[leveloffset=+2]
+
+// Generating a key pair for cluster node SSH access
+include::modules/ssh-agent-using.adoc[leveloffset=+2]
+
+== Preparing for the installation
+
+Before you extend nodes to {aws-service-name-full}, you must prepare certain resources for the cluster installation environment.
+
+// Minimum resource requirements for cluster installation
+include::modules/installation-minimum-resource-requirements.adoc[leveloffset=+2]
+
+// Tested instance types for AWS
+include::modules/installation-aws-tested-machine-types.adoc[leveloffset=+2]
+
+[role="_additional-resources"]
+.Additional resources
+
+ifdef::aws-localzones[]
+* See link:https://aws.amazon.com/about-aws/global-infrastructure/localzones/features/[AWS Local Zones features]
+  in the AWS documentation for more information about {aws-service-name-full} and the supported instances types and services.
+endif::aws-localzones[]
+ifdef::aws-wavelength[]
+* See link:https://aws.amazon.com/wavelength/features[Features][{aws-service-name-full}] 
+  in the AWS documentation for more information about {aws-service-name-full} features.
+endif::aws-wavelength[]
+
+// Creating the installation files for AWS
+//include::modules/installation-user-infra-generate.adoc[leveloffset=+2]
```

Comment:
This is commented out, so it is no longer required?

Reply:
if you check the rendered page, it seems to be duplicated

Reply:
OK. I cannot see the duplication. Am I missing something?